pkg/test/expectations/expectations.go

```diff
@@ -109,7 +125,7 @@ func ExpectCleanedUp(c client.Client) {
 	nodes := v1.NodeList{}
 	Expect(c.List(ctx, &nodes)).To(Succeed())
 	for _, node := range nodes.Items {
-		ExpectDeleted(c, &node)
+		ExpectDeletedNode(c, &node)
```

Comment:
ExpectNodeDeleted sounds better to me.

Reply:
Is there a reason you can't use ExpectDeleted?

Reply:
Our nodes now have the finalizers added with this code, so ExpectDeleted will only call delete, but not patch the finalizer out. 

I was toying with `ExpectNodeDeleted` or `ExpectDeletedNode` and decided that the former sounds like we're not doing any deletion action, but just checking for existence. The latter sounds more like we're doing an action then checking to me.